security/keycloak-authz-reactive/src/test/java/io/quarkus/ts/security/keycloak/authz/reactive/KeycloakAuthzSecurityReactiveIT.java

```diff
@@ -10,8 +14,8 @@
 public class KeycloakAuthzSecurityReactiveIT extends BaseAuthzSecurityReactiveIT {
 
     //TODO Remove workaround after Keycloak is fixed https://github.com/keycloak/keycloak/issues/9916
-    @KeycloakContainer(command = { "start-dev --import-realm --hostname-strict=false" })
-    static KeycloakService keycloak = new KeycloakService("/keycloak-realm.json", REALM_DEFAULT, "/realms")
+    @KeycloakContainer(command = { "start-dev", "--import-realm" })
```

Comment:
Is there reason why `--hostname-strict=false` is  removed when in other cases it stay.

Reply:
yes, I wrote about this (vaguely) in the PR description, I changed it accidentally and it made no difference, so I suppose it doesn't really matter. If it doesn't cause test failure I don't think it needs to be there. Good catch, it was mistake. If it is somehow important for you, I'll change it in next PR when fixing mTLS.

Reply:
If it's work I'm fine with it. Hope that this not cause the problem in future.

Reply:
> If it's work I'm fine with it. Hope that this not cause the problem in future.

IMO tests need to be idempotent, otherwise our job will become very dangerous

Reply:
we will see other integration runs like Windows and Podman... I don't expect there should be difference.